addons/website_payment/static/src/js/website_payment_donation.js

```diff
@@ -22,6 +24,7 @@ publicWidget.registry.WebsitePaymentDonation = publicWidget.Widget.extend({
         if (otherAmountEl) {
             otherAmountEl.checked = true;
         }
+        this._onChangeAmountInput();
```

Comment:
Do we really need this?

Reply:
https://www.loom.com/share/f72dbcf01d1646d6919bf16413dacc13?sid=decf91e2-41e2-4959-8bbd-3998a99e14d5

Yes, if a user selects the custom amount radio button, a warning is displayed. However, if the user directly clicks on the input field, the `_onFocusAmountInput` function changes the radio button to checked, but the warning will still not be displayed. To ensure the warning is displayed when focusing on the input, we need to call the warning function at that time.